gril/grilrequest.c

```diff
@@ -808,25 +808,43 @@ void g_ril_request_set_smsc_address(GRil *gril,
 
 void g_ril_request_dial(GRil *gril,
 			const struct ofono_phone_number *ph,
+			gboolean is_emergency,
 			enum ofono_clir_option clir,
 			struct parcel *rilp)
 {
+	const char *number = phone_number_to_string(ph);
+	int call_domain = is_emergency ? 3 : 1;
 	parcel_init(rilp);
 
 	/* Number to dial */
-	parcel_w_string(rilp, phone_number_to_string(ph));
+	parcel_w_string(rilp, number);
 	/* CLIR mode */
 	parcel_w_int32(rilp, clir);
+	if (g_ril_vendor(gril) == OFONO_RIL_VENDOR_SAMSUNG_MSM_822x) {
+		parcel_w_int32(rilp, 0); // CallDetails.call_type
+		parcel_w_int32(rilp, call_domain); // Normal or emergency call domain
```

Comment:
Line over 80 characters.

Maybe this?

```suggestion
		parcel_w_int32(rilp, call_domain); // Normal or emergency
```

Reply:
This is exactly why I do not like tabs to indent - it will cause confusion with the line length and columns: On my IDE this line below 80 characters because my tab size is 4 spaces. How should I guess this from the code base alone (No knowledge about the original settings)? I can now just _assume_ your tab size might be 8 characters? Or 6? I don't know. Tabs are evil these days.

Reply:
[Tabs are 8 characters, and thus indentations are also 8 characters.](https://www.kernel.org/doc/html/v4.10/process/coding-style.html#indentation).

Reply:
Well I am not going to discuss this further here, and this project is not a part of the Linux kernel, so any rule might or might not apply. I just say I simply cannot know it, so I need to guess. In my company all tabs are 4 characters for example, it is in my DNA so to say. So when I started to fix the overly long lines, I had to do it now 2 times. Any how, its fixed now...

Reply:
[oFono coding style must follow every rule for Linux kernel](https://github.com/ubports/ofono/blob/xenial/doc/coding-style.txt)